src/components/randomize.jsx

```diff
@@ -44,8 +44,15 @@ export default class Randomize extends React.Component {
   }
 
   render() {
-    return (
-      <button className="button" className="btn btn-outline-danger btn-sm" onClick={this.handleClick}>Randomize Colors</button>      
-    )
+    if(this.props.enabled == "enabled"){
+      return (
+        <button className="button" className="btn btn-outline-danger btn-sm" onClick={this.handleClick}>Randomize Colors</button>
+      )
+    }
+    else{
+      return (
+        <button className="button" className="btn btn-outline-danger btn-sm" disabled={ this.props.enabled } onClick={this.handleClick}>Randomize Colors</button>
```

Comment:
@calebhsu, I didn't add in the two classNames. Do you know who did Randomize? I don't want to remove it if the developer put them there on purpose. Bootstrap doesn't allow to disable hover http://getbootstrap.com/css/#buttons. There should have been an "alert" graphic on your mouse pointer when you hover over the buttons. Did you see it?